github/migrations.go

```diff
@@ -104,8 +104,12 @@ func (s *MigrationService) StartMigration(ctx context.Context, org string, repos
 // ListMigrations lists the most recent migrations.
 //
 // GitHub API docs: https://developer.github.com/v3/migration/migrations/#get-a-list-of-migrations
```

Comment:
LGTM, but while you are here, @gauntface, could you please update the outdated URL to this?
https://developer.github.com/v3/migrations/orgs/#get-a-list-of-organization-migrations